src/Security/Authentication/AuthenticatedSessionStateHandler.php

```diff
@@ -18,15 +18,14 @@
 
 namespace Surfnet\SamlBundle\Security\Authentication;
 
-use Surfnet\SamlBundle\Security\Exception\LogicException;
 use Surfnet\SamlBundle\Value\DateTime;
 
 interface AuthenticatedSessionStateHandler
 {
     /**
      * Sets the moment at which the user was authenticated
      *
-     * @throws LogicException when an authentication moment was already logged
+     * @throws \LogicException when an authentication moment was already logged
```

Comment:
Is this what you want? You now throw a less specific exception than we used to

Reply:
Good catch! This was actually wrong.

Also, I found out what was going on with phpcs not detecting uses in comments. So i fixed that as well, as it was causing issues here. 

```
  <rule ref="SlevomatCodingStandard.Namespaces.UnusedUses">
    <properties>
      <property name="searchAnnotations" type="bool" value="true"/>
    </properties>
  </rule>
```